testing/python/language/test_tilelang_language_eager_jit.py

```
@@ -73,7 +74,13 @@ def gemm_ptr(
                 T.gemm(A_shared, B_shared, C_local)
             T.copy(C_local, C[bx * block_M, by * block_N])
 
-    prod = product([T.float16, T.float32], [T.float32])
+    # ROCm WMMA only supports float16/bfloat16 as input; skip float32 input on non-CUDA.
+    try:
+        _is_cuda = target_is_cuda(determine_target("auto", return_object=True))
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)